lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -319,6 +339,18 @@ def test_instructor_level(self):
             # TODO: make these work
             if endpoint in ['update_forum_role_membership']:
                 continue
+            elif endpoint == 'get_problem_responses':
+                mock_problem_key = Mock(return_value=u'')
+                mock_problem_key.course_key = self.course.id
+                with patch.object(UsageKey, 'from_string') as patched_method:
+                    patched_method.return_value = mock_problem_key
+                    self._access_endpoint(
+                        endpoint,
+                        args,
+                        200,
+                        "Instructor should be allowed to access endpoint " + endpoint
+                    )
```

Comment:
This is the same code as L301-312. Refactor into a method (perhaps take a `msg` string as an argument)